site/content/docs/5.2/getting-started/webpack.md

```diff
@@ -1,112 +1,220 @@
+
+```text
+my-project/
+├── dist/
+│   └── index.html
+├── src/
+│   ├── js/
+│   │   └── main.js
+│   └── scss/
+│       ├── _custom.scss
+│       └── styles.scss
+├── package-lock.json
+├── package.json
+└── webpack.config.js
 ```
 
-In this case you may use your existing rule for `css` without any special modifications to webpack config, except you don't need `sass-loader` just [style-loader](https://github.com/webpack-contrib/style-loader) and [css-loader](https://github.com/webpack-contrib/css-loader).
-
-```js
-// ...
-module: {
-  rules: [
-    {
-      test: /\.css$/,
-      use: [
-        'style-loader',
-        'css-loader'
-      ]
-    }
-  ]
-}
-// ...
-```
+At this point, everything is in the right place, but Webpack won't work because we haven't created our `webpack.config.js` file yet.
+
+## Configure Webpack
+
+With dependencies installed and our project folder ready for us to start coding, we can now configure Webpack and run our project locally.
+
+1. **Open `webpack.config.js` in your editor.** Since it's blank, we'll need to add some boilerplate config to it so we can start our server. This part of the config tells Webpack were to look for our project's JavaScript, where to output the compiled code to (`dist`), and how the development server should behave (pulling from the `dist` folder with hot reload).
+
+   ```js
+   const path = require('path')
+
+   module.exports = {
+     entry: './src/js/index.js',
+     output: {
+       filename: 'main.js',
+       path: path.resolve(__dirname, 'dist'),
+     },
+     devServer:{
+       static: path.resolve(__dirname, 'dist'),
+       port: 8080,
+       hot: true
+     }
+   }
+   ```
+
+2. **Next we build our `dist/index.html`.** This is the HTML page Webpack will load in the browser to utilize the bundled CSS and JS we'll add to it in later steps. Before we can do that, we have to give it something to render and include the `output` JS from the previous step.
+
+   ```html
+   <!doctype html>
+   <html lang="en">
+     <head>
+       <meta charset="utf-8">
+       <meta name="viewport" content="width=device-width, initial-scale=1">
+       <title>Bootstrap w/ Webpack</title>
+     </head>
+     <body>
+       <div class="container py-4 px-3 mx-auto">
+         <h1>Hello, Bootstrap and Webpack!</h1>
+         <button class="btn btn-primary">Primary button</button>
+       </div>
+       <script src="./main.js"></script>
+     </body>
+   </html>
+   ```
+
+   We're including a little bit of Bootstrap styling here with the `div class="container"` and `<button>` so that we see when Bootstrap's CSS is loaded by Webpack.
+
+3. **Now we need an npm script to run Webpack.** Open `package.json` and add the following scripts section. This should be in addition to the project name, description, vesion, and dependencies we installed earlier.
+
+   ```js
+   {
+   // ...
+     "scripts": {
+       "start": "webpack serve --mode development"
```

Comment:
```suggestion
       "start": "webpack serve --mode development",
       // ...
```

I propose this modification because this was the content of the scripts part when I installed it:

```
"scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
```